server/src/main/java/ru/hse/server/exception/UserAlreadyExistException.java

```diff
@@ -0,0 +1,7 @@
+package ru.hse.server.exception;
+
+public class UserAlreadyExistException extends RuntimeException {
```

Comment:
Doesn't spring have a corresponding exception?

And perhaps it’s better not to do it specifically for the user, but to do something more general: `EntityAlreadyExistException`.  And describe a specific ('user', `lesson`, `group`) exception message